client/gutenberg/extensions/tiled-gallery/tiled-grid/group.js

```diff
@@ -23,16 +18,4 @@ export class Jetpack_Tiled_Gallery_Group {
 
 		return 1 / ratio;
 	};
-
-	items = ( needs_attachment_link, grayscale ) => {
-		const items = [];
-
-		for ( const image of this.images ) {
-			items.push(
-				new Jetpack_Tiled_Gallery_Rectangular_Item( image, needs_attachment_link, grayscale )
```

Comment:
I was hoping that lack of calling `items()` somewhere was causing layout consisting only squares instead of rectangular but it doesn't look like Item classes are responsible for it:

https://github.com/Automattic/jetpack/blob/e07addf220d02226f04d874b1b565ceb9f502839/modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php#L77